modules/KIWILinuxRC.sh

```diff
@@ -6967,6 +6967,20 @@ function mountSystem {
     return $retval
 }
 #======================================
+# mountOrCopyLiveCD
+#--------------------------------------
+function mountOrCopyLiveCD {
+    if [ -n $TORAM ]; then
+        mkdir -p ${LIVECD}R $LIVECD && eval mount $cdopt $biosBootDevice ${LIVECD}R
+        SIZE="$(/usr/bin/du -s /${LIVECD}R | /usr/bin/gawk '{print int($1*1.1)}')"
+        /bin/mount -t tmpfs -o size=${SIZE}k tmpfs $LIVECD
+        cp -ar ${LIVECD}R/* $LIVECD/
+        umount ${LIVECD}R
```

Comment:
looks good, just add 

```
rmdir ${LIVECD}R
```

and it would be nice to tell the user what we are doing here. So a message saying

```
Echo "Copying CD system into tmpfs"
```

or similar

Thanks